src/error.ts

```diff
@@ -228,6 +228,44 @@ export class MongoParseError extends MongoDriverError {
   }
 }
 
+export class MongoAPIError extends MongoDriverError {
+  constructor(message: string) {
+    super(message);
+    this.name = 'MongoAPIError';
```

Comment:
NODE-3368 *just* got merged which made the name prop of error classes getter functions. Moving forward names should implemented as:
```
get name(): string {
  return 'MongoAPIError';
}
```

Can you add these changes so we can stay consistent?

Reply:
Will do